src/main/java/com/milamberBrass/brass_armory/entities/custom/BAArrowEntity.java

```diff
@@ -62,19 +66,18 @@ public BAArrowEntity(EntityType<? extends AbstractArrowEntity> entityType, World
 	public BAArrowEntity(EntityType<? extends AbstractArrowEntity> type, World worldIn, LivingEntity shooter, ArrowType typeIn) {
 		super(type, shooter, worldIn);
 		this.setArrowType(typeIn.getString());
-		if (ArrowType.byName(this.getDataManager().get(ARROW_TYPE)) == ArrowType.LASER) {
+		if (this.isArrowType(ArrowType.LASER)) {
 			this.setPierceLevel((byte) 5);
 		}
+		this.setDamage(this.getArrowType().getDamage());
```

Comment:
Damage is not yet implemented right?

Reply:
it should be in this build? I thought?

Reply:
It seems to me that we create a new variable called damage and assign our damage to that variable, without ever doing something with it.

(We don't override the already existing one, since it's private I believe)

Reply:
I'll take a look tonight, I thought I had changed the arrow function that asks for the damage to call ours, but I should probably change the name to make it actually call ours instead of the other built-in damage field.